### ISSUE TITLE
We should have an option to ignore nonconflicting redundant lines

### ISSUE THREAD

**Issue by eggrobin** [unicode-org/unicodetools#861]
Our strategy for handling merge conflicts (previously: #444, #568, #583) is to ignore the conflict markers and interpret both branches. This is fine unless there is overlap between modified ranges, as happened to me recently while synchronizing https://github.com/unicode-org/unicodetools/pull/578; in that case, even if the overlap agrees, we need to manually merge ranges, which is error-prone (recall that we expect to have to juggle upwards of 50 of those PRs for a long time).

There are `Merge` objects passed around in PropertyParsingInfo, null in many places; we could replace those nulls with something that accepts consistent values, maybe behind a flag so we do that only when handling merge conflicts.

**Comment by macchiati** [comment]
Good idea. Could even do it for all dev versions that are regenerated, automatically.
